src/commands/dev/index.js

```diff
@@ -175,14 +176,11 @@ const handleLiveTunnel = async ({ flags, site, api, settings, log }) => {
   }
 }
 
-const reportAnalytics = async ({ config, settings }) => {
-  await config.runHook('analytics', {
-    eventName: 'command',
-    payload: {
-      command: 'dev',
-      projectType: settings.framework || 'custom',
-      live: flagsLib.live || false,
-    },
+const reportAnalytics = async ({ settings, flags }) => {
+  await track('command', {
+    command: 'dev',
+    projectType: settings.framework || 'custom',
+    live: flags.live || false,
```

Comment:
I believe I introduces this bug a while back. We should pass in `flags.live` instead of using `flagsLib.live`